.gitignore

```diff
@@ -77,7 +77,7 @@ target/
 .ipynb_checkpoints/
 
 # exclude data from source control by default
-data
+/data/
```

Comment:
I thought it would be cleaner this way. We do not want to ignore all and any file/folder named data. I think the intent was always to only ignore the root data folder.
But I agree I should not try to change things that are unrelated to the PR.